src/robot-trajectory-manager.ts

```diff
@@ -95,9 +97,14 @@ export class DefaultTrajectoryManager {
     return resp as TimeResponse;
   }
 
+  static getRobotNameFromPathId(pathId: number, trajectories: readonly Trajectory[]): string | undefined {
```

Comment:
this shouldn't be needed now that the trajectory contains the robot name.

Reply:
Related to https://github.com/osrf/romi-dashboard/pull/40#discussion_r451169117